go-controller/pkg/ovn/multicast_test.go

```diff
@@ -280,6 +280,104 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Operations with IP Address Family", f
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			})
 
+			ginkgo.It("tests enabling multicast in multiple namespaces with a long name > 42 characters with a pod "+ipModeStr(m), func() {
+				app.Action = func(ctx *cli.Context) error {
+					longNameSpace1Name := "abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijk" // create with 63 characters
+					namespace1 := *newNamespace(longNameSpace1Name)
+					longNameSpace2Name := "abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijl" // create with 63 characters
+					namespace2 := *newNamespace(longNameSpace2Name)
+					nPodTestV4 := newTPod(
```

Comment:
I think you don't need pods in this test at all, we only care about acls, and the port group will be created even if no pods are present?

Reply:
removed pods scenario from the test, resolving comment.